### PR TITLE
ascanrules Beta: ScanRule Consistency

### PR DESCRIPTION
**Pull request by kingthorin**
- Scan Rule class names.
- Unit Test class names.
- Message.properties keys (and some values).
- Help content.
- Remove boilerplate JavaDoc.

Part of: zaproxy/zaproxy#6049

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html [hunk 1]
```diff
@@ -135,160 +135,160 @@ <H2>Proxy Disclosure</H2>
...
 <p>
-Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanner.java">RelativePathConfusionScanner.java</a>
+Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java">RelativePathConfusionScanRule.java</a>
 
 <H2>Remote Code Execution - CVE-2012-1823</H2>
 Detect CVE-2012-1823 to perform Remote Code Execution on a PHP-CGI based web server.
 <p>
-Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RemoteCodeExecutionCVE20121823.java">RemoteCodeExecutionCVE20121823.java</a>
+Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RemoteCodeExecutionCVE20121823ScanRule.java">RemoteCodeExecutionCVE20121823ScanRule.java</a>
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html:150]: Needs update in CVE case.

File: addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html [hunk 2]
```diff
@@ -135,160 +135,160 @@ <H2>Proxy Disclosure</H2>
...
+Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/ShellShockScanRule.java">ShellShockScanRule.java</a>
 
 <H2>Source Code Disclosure - CVE-2012-1823</H2>
 Exploit CVE-2012-1823 to disclose server-side PHP source code on a PHP-CGI based web server.<br>
 Only analyzes responses that are text based (HTML, JSON, XML, etc.), in order to avoid false positives which may occur with image or other binary content.<br>
 JavaScript responses are only anaylzed when a LOW alert threshold is set.
 <p>
-Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureCVE20121823.java">SourceCodeDisclosureCVE20121823.java</a>
+Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureCVE20121823ScanRule.java">SourceCodeDisclosureCVE20121823ScanRule.java</a>
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html:171]: Same here.

File: addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/RemoteCodeExecutionScanRuleCve20121823UnitTest.java [hunk 3]
```diff
@@ -34,13 +34,13 @@
 import org.zaproxy.zap.model.TechSet;
 import org.zaproxy.zap.testutils.NanoServerHandler;
 
-/** Unit test for {@link RemoteCodeExecutionCVE20121823}. */
-public class RemoteCodeExecutionCVE20121823UnitTest
-        extends ActiveScannerTest<RemoteCodeExecutionCVE20121823> {
+/** Unit test for {@link RemoteCodeExecutionCve0121823ScanRule}. */
+public class RemoteCodeExecutionScanRuleCve20121823UnitTest
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/RemoteCodeExecutionScanRuleCve20121823UnitTest.java:38]: The `ScanRule` was moved to the middle.

File: addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureScanRuleCve20121823UnitTest.java [hunk 4]
```diff
@@ -39,20 +39,20 @@
 import org.zaproxy.zap.testutils.NanoServerHandler;
 import org.zaproxy.zap.utils.ZapXmlConfiguration;
 
-/** Unit test for {@link SourceCodeDisclosureCVE20121823}. */
-public class SourceCodeDisclosureCVE20121823UnitTest
-        extends ActiveScannerTest<SourceCodeDisclosureCVE20121823> {
+/** Unit test for {@link SourceCodeDisclosureCve0121823ScanRule}. */
+public class SourceCodeDisclosureScanRuleCve20121823UnitTest
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureScanRuleCve20121823UnitTest.java:43]: Same here.

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/IntegerOverflowScanRule.java [hunk 5]
```diff
@@ -3,6 +3,8 @@
  *
  * ZAP is an HTTP/HTTPS proxy for assessing web application security.
  *
+ * Copyright 2020 The ZAP Development Team
+ *
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/IntegerOverflowScanRule.java:7]: This could be reverted.

**Comment by kingthorin** [comment]
Thanks, tweaked.

**Comment by thc202** [comment]
Thank you!

**Pull request merged**